wesnoth-units.po

```diff
@@ -2533,13 +2541,41 @@ msgid ""
 "the other Wayfarers again. My tale might not be that impressive to them, but "
 "at least it is an improvement over my previous encounter with the wyvern...”"
 msgstr ""
+"Z deníku Balaha, pouštního strážce:\n"
+"\n"
+"„... Dnes to byl šestý den. Šest dní pronásledování skrz poušť bez spánku a "
+"bez odpočinku... ale nevyžaduji ani jedno. Rudooký Roh ukradl z mého stáda "
+"příliš mnoho kusů pro své dravé hodování. Musím se ho zbavit, než karavana "
+"vyhladoví. Sledoval jsem ho tak daleko. A bez ohledu na to, jak daleko "
+"uprchne, ho budu sledovat dál.“\n"
+"\n"
+"„... zatraceně! Ten tvor byl jen pár set kroků daleko, ale byl jsem oslněn "
+"sluncem! Podařilo se mi vstřelit pár šípů do té nepříjemné věci. Ale "
+"posloužilo to jen k tomu, že jsem ho zpomalil. Stále se však zdá, že je až "
+"únavně rychlejší než jsem já. S trochou štěstí ho dostihnu, než se dostane "
+"do hor.“\n"
+"\n"
+"„... jedenáctý den. Poté, co přistál na svahu, jsem tu stvůru zahnal do "
+"kouta. Ale objevili se další tři, s kterými jsem nepočítal. Ptáci Rohové "
+"jsou v přímém zápolení divocí protivníci. Už bojovat s jedním není "
+"jednoduchý úkol, ale se čtyřmi najednou? Mám štěstí, že jsem s sebou měl "
+"druhou sekeru, jinak bych byl během té bitvy těžce raněn. Teď si obvazuji "
+"zraněnou nohu a čekám, až to zabere. Potom se vrátím zabít poslední dva. "
+"Možná si pak konečně najdu čas na jídlo a odpočinek.“\n"
+"\n"
+"„... i pro mě je překvapením, že jsem při své jednadvacetidenní expedici "
+"procestoval téměř polovinu Severní pouště. Karavaně by to trvalo téměř tři "
+"měsíce. Předpokládám ale, že je to tím, že jsem cestoval poněkud nalehko. "
+"Kladem této cesty je, že se mi podařilo obstarat trochu soleného masa z "
+"ptáků Nohů. To by pro děti mohla být dobrá pochoutka. Také bude hezké znovu "
+"vidět některé z ostatních poutníků. Můj příběh na ně možná nebude působit "
+"tak strhujícím dojmem, ale oproti mému předchozímu setkání s wyvernou je to "
+"přinejmenším zlepšení...“"
 
 #. [unit_type]: id=Dune Windbolt, race=dunefolk
 #: data/core/units/dunefolk/Windbolt.cfg:8
-#, fuzzy
-#| msgid "Dune Rider"
 msgid "Dune Windbolt"
-msgstr "Dunový jezdec"
+msgstr "Dunový jízdní obkružovač"
```

Comment:
```suggestion
msgstr "Pouštní jízdní obkružovač"
```

Reply:
Ten název jednotky jsem si tak dlouho neúspěšně rešeršoval, až jsem to Dunový nakonec neopravil.